Power Up Proto/src/org/usfirst/frc/team5493/robot/Robot.java

```diff
@@ -13,7 +13,7 @@
 import org.usfirst.frc.team5493.robot.subsystems.CubeControls;
 import org.usfirst.frc.team5493.robot.subsystems.DriveBase;
 import org.usfirst.frc.team5493.robot.subsystems.OneClimbyBoi;
-import org.usfirst.frc.team5493.robot.subsystems.ThrowDaggersInBensEyes;
+//import org.usfirst.frc.team5493.robot.subsystems.ThrowDaggersInBensEyes;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)